stream_alert/rule_processor/payload.py

```diff
@@ -331,7 +332,19 @@ def pre_parse(self):
             self.raw_record['Sns']['MessageId'],
             self.raw_record['EventSubscriptionArn'])
 
-        self.pre_parsed_record = self.raw_record['Sns']['Message']
+        # check if SNS message is an Amazon S3 Notification type
+        if 'Subject' in self.raw_record['Sns'] and \
```

Comment:
Please instead use a dictionary lookup here to simplify this:

```
if self.raw_record['Sns'].get('Subject') == 'Amazon S3 Notification':
    ... 
```

Also, to be consistent with our current style, can you please switch to using single quotes for string literals instead of double? ie: `'Amazon S3 Notification'` vs the current `"Amazon S3 Notification"`